GitTabExpansion.ps1

```diff
@@ -223,6 +223,11 @@ function GitTabExpansion($lastBlock) {
         "^push.* (?<remote>\S+) (?<ref>[^\s\:]*\:)(?<branch>\S*)$" {
             gitRemoteBranches $matches['remote'] $matches['ref'] $matches['branch']
         }
+        
+        # Handles git push remote +<branch>  (force push)
+        "^push.* (?<remote>\S+) (?<ref>\+)(?<branch>\S*)$" {

```

Comment:
I don't think this is quite what we want... This will capture the `+` in the `ref` group and try matching remote branches based on that.

I _think_ it would be sufficient to add `\+?` above/below before we capture `<ref>:<branch>` or just `<branch>` - give that a try?